app/components/agenda/agendaitem/agendaitem-case/subcase-document/document-link/component.js

```diff
@@ -123,9 +124,17 @@ export default Component.extend(isAuthenticatedMixin, UploadDocumentMixin, MyDoc
       if (uploadedFile) {
         const container = this.get('documentContainer');
         const doc = await this.get('documentContainer.lastDocumentVersion');
-        doc.rollbackAttributes()
+        await doc.rollbackAttributes();
+        // await container.rollbackAttributes();
```

Comment:
changed this, some experimental code was left it seems

Reply:
fixed

Reply:
Is the `await` strictly necessary?  [rollbackAttributes](https://api.emberjs.com/ember-data/3.17/classes/Model/methods/rollbackAttributes?anchor=rollbackAttributes) is a sync call as far as I understand ... ( I suppose there is more going on since you explicitly changed it ...)